NOLF2/ClientShellDLL/ClientShellShared/SFXMgr.cpp

```diff
@@ -278,6 +278,12 @@ void CSFXMgr::HandleSFXMsg(HLOCALOBJ hObj, ILTMessage_Read *pMsg)
 			debris.vPos			= pMsg->ReadCompPos();
 			debris.nDebrisId	= pMsg->Readuint8();
 
+			if (std::isnan(debris.rRot.m_Quat[0]) || std::isnan(debris.rRot.m_Quat[1])
+				|| std::isnan(debris.rRot.m_Quat[2]) || std::isnan(debris.rRot.m_Quat[3]))
```

Comment:
maybe have the logical or at the end of the previous line than at the beginning of the...
it would more clearly convey that the tests are continuing on the next line.
and LTRotation has a `[]` operator to access the m_Quat member instead of accessing it directly:
sdk/inc/ltrotation.h#L133